### PR TITLE
Update branding to 3.1.12

### PR DESCRIPTION
**Pull request by aik-jahoda**

### CODE REVIEW

File: .config/dotnet-tools.json [hunk 1]
```diff
@@ -15,4 +15,4 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
```
Comment (Anipik) [inline .config/dotnet-tools.json:18]: We can correct this one in extra commit

**Review by Anipik — COMMENTED** [review]
we will need to update the packageindex.json file to reflect the new released packages i.e. system.io.pipelines and Microsoft.bcl.hashcode

**Review by Anipik — APPROVED** [review]

**Comment by Anipik** [comment]
```
If you’re reading this, that means the Helix work item you’re trying to find the logs for has dead-lettered.

What this means:

- All attempts to retry execution of this work item were unable to complete.  This can be both for infrastructure reasons (problems within Azure) or issues with the work item (for instance, causing a machine to reboot unexpectedly or killing the Helix client on the machine will force a retry).
- No further work will be done for this specific work item, and its exit code is set to an artificial -1 (since it did not complete, there is no real exit code).

Common causes:

- Disabled queue (end-of-life Helix queues are automatically forwarded to deadletter and will fail instantly)
- Unhealthy Helix Client machine(s)
- Azure issues (e.g. Service Bus is overloaded)
- Malformed payloads; if Helix cannot download and unzip all payloads successfully, work will retry until dead-lettered.

For follow up:

- Check if your Helix Queue is still enabled, either via the metadata you see by browsing to https://helix.dot.net/api/info/queues?api-version=2019-06-17 or recent emails from the .NET Engineering Infrastructure team.
- Check that all work item payloads are accessible using a browser.
- If you are sending to a non-disabled queue and find this error repeatedly occurring, please contact the dnceng team.
- If a single, specific work item dead letters and others do not, consider local debugging; it may be causing spontaneous reboot (or trigging one intentionally).
```

@MattGal we are hitting this on the redhat any idea why ?

**Review by mmitche — APPROVED** [review]

**Comment by MattGal** [comment]
> ```
> If you’re reading this, that means the Helix work item you’re trying to find the logs for has dead-lettered.
> 
> What this means:
> 
> - All attempts to retry execution of this work item were unable to complete.  This can be both for infrastructure reasons (problems within Azure) or issues with the work item (for instance, causing a machine to reboot unexpectedly or killing the Helix client on the machine will force a retry).
> - No further work will be done for this specific work item, and its exit code is set to an artificial -1 (since it did not complete, there is no real exit code).
> 
> Common causes:
> 
> - Disabled queue (end-of-life Helix queues are automatically forwarded to deadletter and will fail instantly)
> - Unhealthy Helix Client machine(s)
> - Azure issues (e.g. Service Bus is overloaded)
> - Malformed payloads; if Helix cannot download and unzip all payloads successfully, work will retry until dead-lettered.
> 
> For follow up:
> 
> - Check if your Helix Queue is still enabled, either via the metadata you see by browsing to https://helix.dot.net/api/info/queues?api-version=2019-06-17 or recent emails from the .NET Engineering Infrastructure team.
> - Check that all work item payloads are accessible using a browser.
> - If you are sending to a non-disabled queue and find this error repeatedly occurring, please contact the dnceng team.
> - If a single, specific work item dead letters and others do not, consider local debugging; it may be causing spontaneous reboot (or trigging one intentionally).
> ```
> 
> @MattGal we are hitting this on the redhat any idea why ?

Hi @Anipik,

This would be the **"end-of-life Helix queues are automatically forwarded to deadletter and will fail instantly"** aspect of the message you pasted.  Redhat 6 was end of life as of November 30, 2020.  Its removal was telegraphed via email at least a week before.  We try to remove helix queues for non-supported OSes to save money and regain total core capacity for other usage.  When you see this for every work item in a run where other OSes did not fail, I recommend checking with your favorite search engine for whether the operating system in question is still in support.

If you have any questions about how to remove the queue from usage, please feel free to reach out via Teams / Email.

**Comment by Anipik** [comment]
Thanks for the info @MattGal

**Pull request merged**